### PR TITLE
Add docs for seed:run --specific

### PR DESCRIPTION
**Pull request by felixmosh**
related to [3335](https://github.com/tgriesser/knex/pull/3335)

### CODE REVIEW

**Comment by kibertoad** [comment]
@felixmosh Thank you! Could you please remove index.html from this PR? We rebuild it ourselves before releases, otherwise it's a conflict hell :D

**Comment by felixmosh** [comment]
Done

File: sections/seeds.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ export default [
     content: [
       "`directory`: a relative path to the directory containing the seed files (default `./seeds`)",
       "`loadExtensions`: array of file extensions which knex will treat as seeds. For example, if you have typescript transpiled into javascript in the same folder, you want to execute only javascript seeds. In this case, set `loadExtensions` to `['.js']`  (Notice the dot!)  (default `['.co', '.coffee', '.eg', '.iced', '.js', '.litcoffee', '.ls', '.ts']`)",
+      "`specific`: a specific seed file to run from the seeds directory (default `undefined`)"
```
Comment (kibertoad) [inline sections/seeds.js]: might be helpful to clarify that if nil, all seeds are run.

Comment (felixmosh) [reply sections/seeds.js]: done

**Pull request merged**